src/presets/firebase/preset.ts

```diff
@@ -68,4 +77,37 @@ const firebase = defineNitroPreset(
   }
 );
 
-export default [firebase] as const;
+const firebaseAppHosting = defineNitroPreset(
+  {
+    extends: "node-server",
+    serveStatic: true,
+    hooks: {
+      async compiled(nitro) {
+        await writeFile(
+          join(nitro.options.rootDir, ".apphosting/bundle.yaml"),
+          stringifyYAML({
+            version: "v1",
+            runConfig: {
+              runCommand: "node .output/server/index.mjs",
+              ...(nitro.options.firebase as AppHostingOptions)?.appHosting,
+            },
+            metadata: {
+              framework: nitro.options.framework.name || "nitropack",
```

Comment:
Is it possible to have this pull the name of any frameworks that are built on top of nitro and fall back to nitro if they don't exist? For example if its a Nuxt application we probably would want "nuxt" to be the framework here

Reply:
Yes, nuxt sets it to `nuxt` (that's why it is configurable)